ui/src/export.tsx

```diff
@@ -386,6 +386,9 @@ function Preview(props: PreviewProps) {
             message: "EXPORT",
             resources: {
               allParticipantsCohort: props.allParticipantsCohort,
+              hasWGSFeatureSet: filteredFeatureSets.some((fs) =>
+                fs.predefinedCriteria.includes("_short_read_wgs")
+              ),
```

Comment:
@tjennison-work This is a quick solution for notifying the workbench that an export would need a VCF extraction but may be too AoU-specific. Let me know if this needs to be changed

Reply:
How about including the list of predefined criteria instead of searching for a particular one so it could be used for any special exports:
```
predefinedCriteria: [...new Set(filteredFeatureSets.map((fs) => fs.predefinedCriteria).flat())],
```

Reply:
That makes more sense, good call